intro-to-git/intro-to-git-vocab-and-concepts.md

```diff
@@ -0,0 +1,127 @@
+
+## Each Repository is a Distinct "Project" Code Base
+
+Git has the concept of a **repository,** or repo. A **repo** is a way to describe a project codebase that is being versioned Git.
+
+Often, a repo will directly correlate to a folder on your machine that contains all of a project's files.
+
+When we say to look at someone else's repo, it likely means that they have their own project and codebase that is being versioned with Git.
+
+## Software is Built as a History
+
+When we compare different versions of code to each other, we see how the code shrinks, grows, and changes between them.
+
+In a sense, version control systems let us understand that software is built over time.
+
+In order to use Git to compare versions, sometimes we will need to tell Git to _track_ different files in a project. This is Git's way of saying that it needs to watch for changes in certain files. Of course we will want Git to watch for changes; that's how we get all the benefits!
+
+### Commits are a Snapshot of History
+
+**A commit** is a representation of a version of code at one point of time.
+
+We will build commits as we build software; we will create commits, and over time, these commits will represent different versions of code and build a history.
+
+Commits not only represent a version of code, but also they contain a lot of extra information. Every commit can also contain the following information:
+
+- A summary of lines of code that were added, deleted, or changed, compared to the last commit
+- A way to refer to the last commit, or its "parent" commit
+- A unique ID for the commit, often called the commit hash
+- A commit message, which will be a string that describes the commit's contents
+- The date and time that the commit was made
+
+Large projects will be made of hundreds or thousands of commits.
+
+## Teams Collaborate by Merging Histories
+
+In a work day, we want any number of programmers to be able to write code, contribute commits, and build software.
+
+Git is a version control system that allows for team collaboration. How?
+
+Collaboration in this sense means that everyone can contribute changes to code, and look at those changes of code.
+
+Every codebase that a team will collaborate on will have a **repository,** or a project folder that contains the files, versioned with Git. (This repository will likely be hosted on github.com)
+
+Every developer who wants to contribute code will need access to the code and files, the repo's commit history, and a way to reference the team's repo. In order to get these three things, every developer will **clone that repo** onto their own machine.
+
+When a developer clones the repo onto their machine, it will create a project folder on their computer, with all of the files inside of it. Then, each developer will write code, run tests, build features, **get updates from the team**, and **make commits.**
+
+Then, each dev who has built commits will send the commits to the team's repository.
```

Comment:
```suggestion
Each dev who makes commits will eventually send the commits to the team's repository.
```

Same as above-- I want this intro to feel more like a storybook that they can memorize, rather than learning words.

Reply:
The surrounding steps use future tense, so how about 'eventually will send'?  I had to add a new separate comment for the suggest directive.